gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py

```diff
@@ -265,8 +285,8 @@ def _fake_resource_data(self):
                     'stitching_cidr': '192.168.0.0/28',
                     'destination_cidr': '192.168.0.0/28',
                     'stitching_mac': '00:0a:95:9d:68:16',
-                    'provider_interface_index': 'provider_interface_index',
-                    'stitching_interface_index': 'stitching_interface_index',
+                    'provider_interface_index': '2',
```

Comment:
Are these values taken just for example? We removed the hardcoded valued form orchestrator right?